internal/display/apps.go

```diff
@@ -106,28 +106,28 @@ func (v *applicationView) Object() interface{} {
 func (r *Renderer) ApplicationList(clients []*management.Client, revealSecrets bool) {
 	resource := "applications"
 
-	r.Heading(fmt.Sprintf("%s (%v)", resource, len(clients)))
-
-	if len(clients) == 0 {
-		r.EmptyState(resource)
-		r.Infof("Use 'auth0 apps create' to add one")
-		return
-	}
-
-	var res []View
+	var appsToRender []View
 	for _, c := range clients {
-		if auth0.StringValue(c.Name) == deprecatedAppName {
+		if auth0.StringValue(c.Name) == deprecatedGlobalAppName {
 			continue
 		}
```

Comment:
I would consider actually updating all the calls to list clients to have `management.Parameter("is_global", "false")` and remove this check completely. Wdyt? 🤔

Reply:
Yeah, probably better way of doing it. Changed.